lib/components/app/responsive-webapp.js

```diff
@@ -98,15 +100,17 @@ class ResponsiveWebapp extends Component {
         setLocationToCurrent({ locationType: 'from' }, intl)
         setMapCenter(map, pt)
       }
-    } else if (query.from && query.to) {
-      map?.fitBounds([query.from, query.to], {
-        duration: 500,
-        padding: getFitBoundsPadding(map, 0.2)
-      })
-    } else if (query.from && !query.to) {
-      setMapCenter(map, query.from)
-    } else if (query.to && !query.from) {
-      setMapCenter(map, query.to)
+    } else if (mainPanelContent === null && autoFly !== false) {
```

Comment:
So this would mean `autoFly` is by default turned on, and you'd have to manually set it to `false` in the config? Just confirming that's what we want. 

I think I'd remove the `!== false` and go with the more conservative map movement by default.

Reply:
Yes default is on. Too many implementers of otp-rr that do not want to see the behavior changed. Better to avoid a breaking change